.buildkite/hooks/pre-command

```diff
@@ -7,6 +7,16 @@
 #
 (
   d=$HOME/cargo-target-cache/"$CARGO_TARGET_CACHE_NAME"
+
+  if [[ -d $d ]]; then
+    du -hs "$d"
+    read -r cacheSizeInGB < <(du -s  --block-size=1000000000 "$d")
+    if [[ $cacheSizeInGB -gt 5 ]]; then
+      echo "$d has gotten too large, removing it"
+      rm -rf "$d"
```

Comment:
oof!

Reply:
which ci machine has a big'un?  I wanna snoop around a little...

Reply:
They all had caches in the 100s of GBs.  I actually manually recreated most of the CI machines though too so right now everything is zero-ed out -- was seeing other networking/disk weirdness